worker/worker-lib.ts

```diff
@@ -0,0 +1,59 @@
+import { v4 as uuid } from 'uuid';
```

Comment:
Not at all wedded to pulling in this dep to keep track of promises, just part of the proof-of-concept nature of what's currently in this PR.

Reply:
Totally, I'd prefer to keep this lib as dependency-free as possible (mainly for google-internal compatibility reasons) - I usually just use a start-at-one-and-increment id generating system for stuff like this.